tests/cephfs/cephfs_system/cephfs_systest_sv_clone_ops.py

```diff
@@ -0,0 +1,395 @@
+                    io_proc_check_status,
+                    io_test,
+                    run_time,
+                    sv_cnt,
+                    clone_cnt,
+                    log_path,
+                    clients,
+                    fs_system_utils,
+                    cephfs_config,
+                ),
+            )
+            p.start()
+            write_procs.append(p)
+        for write_proc in write_procs:
+            write_proc.join()
+        for proc_status in proc_status_list:
+            if proc_status.value == 1:
+                log.error(f"{io_test} failed")
+                io_test_fail = 1
+        if io_test_fail == 1:
+            return 1
+
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Stop logging the Ceph Cluster Cluster status to a log dir")
+
+
+def io_test_runner(
+    io_proc_check_status,
+    io_test,
+    run_time,
+    sv_cnt,
+    clone_cnt,
+    log_path,
+    clients,
+    fs_system_utils,
+    cephfs_config,
+):
+    io_tests = {
+        "sv_test_workflow_1": sv_test_workflow_1,
+        "clone_test_workflow_2": clone_test_workflow_2,
+    }
+    for i in cephfs_config:
+        if "CLUS_MONITOR" not in i:
+            fs_name = i
+
+    if io_test == "sv_test_workflow_1":
+        client = random.choice(clients)
+        io_proc_check_status = io_tests[io_test](
+            run_time, fs_name, sv_cnt, log_path, client, fs_system_utils
+        )
+    else:
+        sv_info_list = []
+        sv_name_list = []
+        k = 0
+
+        while len(sv_name_list) < 10 and k < 20:
+            sv_info = fs_system_utils.get_test_object(cephfs_config, "shared")
+            for i in sv_info:
+                sv_name = i
+            if sv_name not in sv_name_list:
+                sv_info_list.append(sv_info)
+                sv_name_list.append(sv_name)
+            k += 1
+        client = random.choice(clients)
+        io_proc_check_status = io_tests[io_test](
+            run_time,
+            fs_name,
+            clone_cnt,
+            log_path,
+            client,
+            fs_system_utils,
+            sv_info_list,
+        )
+
+    log.info(f"{io_test} status after test : {io_proc_check_status}")
+    return io_proc_check_status
+
+
+def sv_test_workflow_1(run_time, fs_name, sv_cnt, log_path, client, fs_system_utils):
+    log_name = "sv_create_delete_in_bulk"
+    log1 = fs_system_utils.configure_logger(log_path, log_name)
+    log1.info(f"Start {log_name}")
+    cmd = f"ceph fs subvolumegroup ls {fs_name} --f json"
+    out, _ = client.exec_command(sudo=True, cmd=cmd)
+    output = json.loads(out)
+    group_name = output[0]["name"]
+    sv_rm_list = []
+    for k in range(0, sv_cnt):
+        sv_name = f"systest_sv_{k}"
+        sv_obj = {
+            "sv_name": sv_name,
+        }
+        cmd = f"ceph fs subvolume create {fs_name} {sv_name} {group_name}"
```

Comment:
For subvolume Operations can we use common functions which we have in Utils file . if there are any changes we can address at single point. 
Same for clone operations as well

Reply:
The intent here is to invoke create and delete commands in parallel. Relying on test_libs for these may add delays due to additional implementations in testlib module like param validations, hence decided to run them instantly.
Will try using lib modules instead and see if parallelism can still be achieved.

Reply:
+1 to reusing common functions. The intent of adding common functions in libraries is for reuse. Encourage to use them going forward.